rllib/examples/learners/mog_learner.py

```diff
@@ -0,0 +1,134 @@
+"""Example of how to write a custom loss function, custom rl_module and custom config.
+
+This example shows:
+    - how to subclass an existing (torch) Learner and override its `compute_loss_for_module()` method
+    - how you can add your own loss to a custom (or non-custom) rl_module using compute_loss_for_module
+    - how to add add custom pipeline components to a custom config class using build_learner_connector
+    - how to plug in a custom Learner, rl_module, its loss, and use a custom Config to train with
+
+See the :py:class:`~ray.rllib.examples.learners.classes.mog_loss.PPOTorchLearnerCustomMOGLoss`  # noqa
+class for details on how to override the main (PPO) loss function.
+
+This script will create a new critic network that will be a distribution of gaussians to allow for more expressivity
```

Comment:
Ah here comes some explanation of it. Put this above the "this example: ... ", it is more important.